tools/resourcedocsgen/README.md

```diff
@@ -0,0 +1,19 @@
+# Resource Docs generator driver
+
+This tool generates the provider schema using `tfgen`, then calls the docs generator tool in `pulumi/pulumi` to use the schema to generate docs.
+
+## Generating docs
+
+Run `make resource_docs`.
+
+### Using the latest templates in `pulumi/pulumi`
+
+This tool indirectly depends on the `pulumi/pulumi` repo, namely the `pkg/codegen/docs` generator and the templates it uses, which is the underlying tool responsible for generating docs from the Pulumi schema.
+
+To make changes to the templates or to use the latest ones from `master`, you will need to override the module path for `pulumi/pulumi` in your local copy of `pulumi-terraform-bridge` using:
```

Comment:
Is this right? It looks like `gen_resource_docs.sh` actually looks for `../pulumi`, and when I ran it, I got the changes I'd made locally.

Reply:
Ah yes. Good catch. I need to update this instruction. The script regenerates the templates bundle each time. So local modifications can be picked-up easily. I'll update the README. Thank you, @cnunciato, for spotting it!